cmd/node/operationModes.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import "fmt"
+
+const (
+	operationModeImportDb           = "import-db"
+	operationModeFullArchive        = "full-archive"
+	operationModeDbLookupExtension  = "db-lookup-extension"
+	operationModeHistoricalBalances = "historical-balances"
+)
+
+func checkOperationModes(modes []string) error {
+	// db lookup extension and historical balances
+	isInvalid := sliceContainsBothElements(modes, operationModeHistoricalBalances, operationModeDbLookupExtension)
+	if isInvalid {
+		return fmt.Errorf("operation-mode flag cannot contain both db-lookup-extension and historical-balances")
+	}
+
+	return nil
+}
+
+func sliceContainsBothElements(elements []string, first string, second string) bool {
```

Comment:
Here, it's all right to call `sliceContainsElement` twice - shorter implementation (no actual performance issue on short arrays).

Reply:
This function doesn't have a unit test.

Reply:
refactored and added unit tests